tardis/io/logger/colored_logger.py

```diff
@@ -28,11 +28,16 @@ class ColoredFormatter(logging.Formatter):
     Custom logger class for changing levels color
     """
 
+    non_debug = "[$BOLD{name:20s}$RESET][{levelname:18s}]  \n\t{message:s} ($BOLD{filename:s}$RESET:{lineno:d})"
+    debug = "[$BOLD{name:20s}$RESET][{levelname:18s}]  {message:s} ($BOLD{filename:s}$RESET:{lineno:d})"
```

Comment:
Added the formatting inside the `ColoredFormatter` Class so that it can be accessed for the log messages. For both these messages, the `formatter_message` function is called for adding the relevant colours.

Reply:
Shouldn't these be `self` variables inside `__init__`?

Reply:
Yes, this seems to be a better way to proceed. I have restructured the `colored_logger.py` to adhere to this change.